integrations-service/integrations/providers.py

```diff
@@ -172,7 +172,7 @@
             output=BrowserbaseExtensionOutput,
         ),
         BaseProviderMethod(
-            method="get_connect_url",
+            method="get_sesion_connect_url",
```

Comment:
this is incorrect (misspelt). also this should be renamed to `get_connect_url` and same with `get_session_live_urls`.

Also need to be corrected in `integrations-service/integrations/utils/integrations
/browserbase.py`

See: https://github.com/julep-ai/julep/blob/e3b7685b9e5c5f28db3a035c10979a7e27095c9f/typespec/tools/browserbase/main.tsp#L82